includes/wc-template-functions.php

```diff
@@ -1877,6 +1878,8 @@ function woocommerce_form_field( $key, $args, $value = null ) {
 		$args['maxlength'] = ( $args['maxlength'] ) ? 'maxlength="' . absint( $args['maxlength'] ) . '"' : '';
 
 		$args['autocomplete'] = ( $args['autocomplete'] ) ? 'autocomplete="' . esc_attr( $args['autocomplete'] ) . '"' : '';
+		
+		$args['autofocus'] = ( true == esc_attr( $args['autofocus'] ) ) ? apply_filters( 'woocommerce_form_field_autofocus', ' autofocus' ) : '' ;
```

Comment:
esc_attr should not be used for comparison because it is used for sanitizing output.